app/renderer/js/main.js

```diff
@@ -343,6 +356,14 @@ class ServerManagerView {
 				isActive: () => {
 					return this.functionalTabs[tabProps.name] === this.activeTabIndex;
 				},
+				switchLoading: (loading, url) => {
```

Comment:
Do we require this in functional tabs? I think we are not showing loading in settings part.